### DISCUSSION TITLE
Bug!!! when opening my home page

### DISCUSSION THREAD

**Discussion by yosef199** [yasserfarouk/scml#92] [collusion-open-discussion]
![image](https://user-images.githubusercontent.com/44501445/166154568-3d19b377-987a-459c-82ab-684903e45fc7.png)

^^got this^^

**Comment by yasserfarouk** [comment]
Sorry for this issue. Which page were you trying to access on the website?

**Reply by yosef199** [reply]
Still got that issue.
The url is : https://scml.cs.brown.edu/display/id=110

**Comment by yasserfarouk** [comment]
Will return to you soon. In the worst case, you may need to register once more. We had a bug that caused incomplete registrations sometimes and it was resolved but your account may have been affected.

**Reply by yosef199** [reply]
I have no problem with register again, but because I already used both of my mails to
register (and got bugs in both of them) Im asking you to delete my accounts so I could use my mails
to register once more.
My mails are : yosef.zumer@mail.huji.ac.il and yosefzumer@gmail.com
URLs to the homepages:
https://scml.cs.brown.edu/display/id=110
https://scml.cs.brown.edu/display/id=111

**Reply by yasserfarouk** [reply]
Done. You should be able to register again with either email.

**Reply by yosef199** [reply]
Work!
Thank for the help

**Reply by yosef199** [reply]
Can you please also delete my partner account so she could use her mails
to register once more.
email : 
michaela.levi20@gmail.com
url:
https://scml.cs.brown.edu/display/id=119

**Comment by yasserfarouk** [comment]
Done but you do not need to have two registrations for your team. In the end, the whole team must submit one agent using one account. It is not allowed to submit two agents that are developed in collaboration. Please check the game rules in the CFP for details.

thanks
